lib/pymedphys/_experimental/streamlit/apps/wlutz/_excel.py

```diff
@@ -23,7 +23,7 @@
 from pymedphys._imports import streamlit as st
 from pymedphys._imports import xlsxwriter
 
-from . import _utilities
+from pymedphys._experimental.streamlit.utilities import iteration as _iteration
```

Comment:
Why is `_iteration` prepended with an underscore?

Reply:
It's a pattern I have slowly began to adopt. I saw it when I was making commits to the Streamlit codebase:

https://github.com/streamlit/streamlit/blob/df51f926eb4aef81520a9fc9f138ec9514b3b006/lib/streamlit/__init__.py#L64-L70

But essentially, any import that isn't preceded with an `_` is essentially defacto part of the API of that module since it doesn't have the "private flag" of the underscore. I haven't been following it strictly... more just dipping my toes in the water of the idea, testing it out.

Reply:
Gotcha